task_1.js

```diff
@@ -6,4 +6,17 @@ Write a function min that takes two arguments and returns their minimum.
 ჩვენ ვისწავლეთ სტანდარტული ფუნქცია Math.min, რომელიც აბრუნებს უმცირეს არგუმენტს.
 ახლა შეგვიძლია მსგავსი ჩვენით ავაწყოთ.
 დაწერეთ ფუნქცია min რომელიც იღებს 2 არგუმენტს და აბრუნებს მათგან მინიმუმს.
-*/
\ No newline at end of file
+*/
+
+
+function min (a,b){
+    if(a>b){
+        console.log(b);
+    }else if(b>a){
+        console.log(a);
+    }else if(a==b){

```

Comment:
სფეისები და მკაცრი ტოლობა!
```suggestion
    } else if(a === b){

```